WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			independent = yes
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
```

Comment:
When does the AI rise up? What are you referring to? The AI and player can form a custom kingdom. Do you have any instances of when this caused issues? Dejure is dynamic in this period, remember that.